source/_integrations/mealie.markdown

```diff
@@ -46,6 +47,10 @@ The integration will create a calendar for every type of meal plan:
 
 The integration will create a to-do list for every Mealie shopping list.
 
+## Sensors
+
+The integration provides sensors for the statistics like the number of recipes, categories, tags, tools and users.
```

Comment:
```suggestion
The integration provides the following sensors for the statistics:
- number of recipes
- categories (such as beverage, dessert, Italian, seafood, vegetarian)
- tags (such as "alcohol")
- tools
- users
```

Reply:
Oh that's nice, haven't thought about describing it like that